### DISCUSSION TITLE
How to change the value of the react-select

### DISCUSSION THREAD

**Discussion by jcubic** [JedWatson/react-select#5111] [Q&A]
This is like 1 basic thing that you want to do. But I can't find this information anywhere.

How can I change the value of the react-select? I only see defaultValue but it doesn't change the select value when it changes after the initial render.

PS: The documentation is very poor, the API show `setValue: (ValueType, ActionTypes) => void,` but this is a prop and don't show how to use it. The API documentation is no better than typescript definitions.

**Comment by jcubic** [comment]
Found this issue https://github.com/JedWatson/react-select/issues/4592 that is the only documentation for this behavior that was really hard to find. Why not document how to change the value of the component.

**Comment by manjushsh** [comment]
@jcubic, react-select GitHub page has [Example](https://github.com/JedWatson/react-select#with-react-component)
